tests/test_bonfire.py

```diff
@@ -164,18 +164,18 @@ def test_ns_list_option_mine(mocker, caplog, namespace_list: list, reservation_l
 
     actual = " ".join(result.output.split())
 
-    assert " ".join(["namespace-1", "true", "false", "none", "user-1"]) in actual
-    assert " ".join(["namespace-2", "true", "false", "none", "user-2"]) not in actual
-    assert " ".join(["namespace-3", "false", "ready", "none"]) not in actual
-    assert " ".join(["namespace-4", "false", "ready", "none"]) not in actual
-    assert " ".join(["namespace-5", "true", "false", "none", "user-5"]) not in actual
+    assert " ".join(["ns-1", "true", "false", "none", "user-1"]) in actual
+    assert " ".join(["ns-2", "true", "false", "none", "user-2"]) not in actual
+    assert " ".join(["ns-3", "false", "ready", "none"]) not in actual
+    assert " ".join(["ns-4", "false", "ready", "none"]) not in actual
+    assert " ".join(["ns-5", "true", "false", "none", "user-5"]) not in actual
 
 
 def test_ns_list_flag_output(
     mocker,
     caplog,
     namespace_list: list,
-    reservation_list: list,
+    reservation_list: list
```

Comment:
Did this come in as a part of a format? We shouldn't change this.

Reply:
Oh no it didn't sorry. I'll fix that.